Makefile

```
@@ -103,3 +103,10 @@ build-postgres-local:
 	-I./proto/ \
 	-I./third_party/proto/ \
 	example/postgres_arrays/postgres_arrays.proto --gorm_out="engine=postgres,enums=string,gateway:./example/postgres_arrays" --go_out=./example/postgres_arrays
+
+.PHONY: mod
+mod:
+	cd types && go mod tidy
+	cd example && go mod tidy
+	go mod tidy
```

You can future proof this with
```suggestion
	find . -name go.mod -execdir go mod tidy \;
```